doc/version-history.rst

```diff
@@ -4,6 +4,12 @@
 Version History
 ===============
 
+v0.3.1
```

Comment:
Please update the version to `v0.4.0`. The change to ts_tcpip is quite large, don't you think?

Reply:
Agree.